scripts/Create_all_tables_update.sql

```diff
@@ -0,0 +1,277 @@
+-- update tables in one script
+
+-- create index on geometries tables
+CREATE INDEX IF NOT EXISTS geometry_aduid_pruid_idx ON boundaries."Geometry_ADAUID"("ADAUID");
```

Comment:
At some point, we may want to consider creating the indices after the data is loaded. Might not be practical in all cases though.